frontend/src/components/models.ts

```diff
@@ -97,7 +97,15 @@ export interface CnsQueryResponse {
 
 // Relayer types
 export type ApiError = { error: string };
-export type TokenListResponse = TokenList | ApiError;
+export interface TokenInfoWithMinSendAmount extends TokenInfo {
+  minSendAmount: string;
+}
+// Omit the TokenList.tokens type so we can override it with our own.
+export interface TokenListWithMinSendAmount extends Omit<TokenList, 'tokens'> {
+  nativeTokenMinSendAmount: string;
```

Comment:
Should the backend just return the native token info and min send amount just like any other token, instead of special casing it? Right now in `wallet.ts` we manually extend the array of tokens to add the native token, but that doesn't seem necessary/ideal

Reply:
We have to because that's what the backend is currently returning. Until the backend is updated (which I plan to do later) this is what we need to use

Reply:
Are we tracking this in an issue somewhere, either here or in the backend?

Reply:
Just created [this issue](https://github.com/ScopeLift/umbra-api/issues/51) on the API side to track it

Reply:
Looks great, thanks @apbendi